javascript-modal/main.js

```diff
@@ -0,0 +1,13 @@
+const $openButton = document.querySelector('.open-button');
+const $noButton = document.querySelector('.no-button');
+const $modalContainer = document.querySelector('.close');
+
+console.log($modalContainer);
```

Comment:
Delete all console logs before submitting! They are only for testing / debugging purposes. Unless we specifically ask for them in the instructions!